tests/storage-disks-vm

```diff
@@ -130,6 +130,10 @@ if hasNeededAPIExtension disk_io_bus; then
   lxc config device add v1 nvme-ssd disk source="${loopdev}" io.bus=nvme
   lxc start v1
   waitInstanceReady v1
+  if ! lxc exec v1 -- sh -c 'command -v lspci'; then
+    lxc exec v1 -- apt-get update
+    lxc exec v1 -- apt-get install -y pciutils
```

Comment:
Should we be using --no-install-recommends by default on all apt install calls nowadays?